website/docs/r/dx_private_virtual_interface.html.markdown

```diff
@@ -40,6 +42,7 @@ The MTU of a virtual private interface can be either `1500` or `9001` (jumbo fra
 * `dx_gateway_id` - (Optional) The ID of the Direct Connect gateway to which to connect the virtual interface.
```

Comment:
Please mention the new parameter `ipv6_cidr_block_network_border_group ` & that `assign_generated_ipv6_cidr_block` is **required** to set it